### PR TITLE
VMC Support

### PR DESCRIPTION
**Pull request by Louka3000**
- Send shareable trackers and all UnityBones (that we have) through VMC.
- Receive VMC trackers and bones
- Read VRM

~~Needs: https://github.com/SlimeVR/SolarXR-Protocol/pull/102~~
Closes: https://github.com/SlimeVR/SlimeVR-Server/issues/3

### CODE REVIEW

File: server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java [hunk 1]
```diff
@@ -135,10 +142,15 @@ public boolean userEditable() {
...
 	@Override
 	public void dataTick() {
+		timeAtLastUpdate = System.currentTimeMillis();
 	}
 
 	@Override
 	public void tick() {
+		if (useTimeout) {
+			if (System.currentTimeMillis() - timeAtLastUpdate > TIMEOUT_MS)
+				setStatus(TrackerStatus.DISCONNECTED);
```
Comment (Eirenliel) [inline server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: add the reverse of it? it'll always be disconnected

**Review by Eirenliel — CHANGES_REQUESTED** [review]

File: server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java (hunk 1 above)
Comment (Eirenliel) [reply server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: The tracker will never stop being disconnected and will be set to DISCONNECTED again on each tick.

Comment (Louka3000) [reply server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: timeAtLastUpdate gets updated to System.currentTimeMillis, so if it's receiving data it'll be for example:
19348 - 19348 > 3000
0 > 3000
false

If not, it'll be, for example:
29348 - 19348 > 3000
10000 > 3000
true

Comment (Eirenliel) [reply server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: Okay but where does it get out of the disconnected state?

Comment (Louka3000) [reply server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: Trackers received from VMC and feeder app are changed to OK on each data update, but I'll double check HMD later

Comment (Louka3000) [reply server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: Yes HMD sets it to OK... I remembered which part of my code did that and saw you reviewed it asking for a comment :p

Comment (Louka3000) [reply server/src/main/java/dev/slimevr/tracking/trackers/ComputedTracker.java:152]: Resolving this, will rewrite trackers to be better as my next big PR anyway...

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**